scripts/zones/Arrapago_Reef/IDs.lua

```diff
@@ -20,39 +20,35 @@ zones[xi.zone.ARRAPAGO_REEF] =
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
-        FISHING_MESSAGE_OFFSET        = 7057, -- You can't fish here.
-        STAGING_GATE_CLOSER           = 7317, -- You must move closer.
-        STAGING_GATE_INTERACT         = 7318, -- This gate guards an area under Imperial control.
-        STAGING_GATE_ILRUSI           = 7323, -- Ilrusi Atoll Staging Point.
-        CANNOT_LEAVE                  = 7328, -- You cannot leave this area while in the possession of <keyitem>.
-        RESPONSE                      = 7337, -- There is no response...
-        ASSAULT_START_OFFSET          = 7454, -- Max MP Down removed for <name>.
-        DOOR_IS_LOCKED                = 7466, -- The door is locked.  You might be able to open it with <item>.
-        DOOR_IS_LOCKED2               = 7467, -- The door is locked.  You might be able to open it with <item> or <item>.
-        KEY_BREAKS                    = 7468, -- The <item> breaks!
-        YOU_UNLOCK_DOOR               = 7471, -- You unlock the door from the inside.
-        LOCK_SUCCESS                  = 7472, -- <name> successfully opened the lock with the <item>!
-        LOCK_FAIL                     = 7473, -- <name> failed to open the lock with the <item>...
-        PARTY_MEMBERS_HAVE_FALLEN     = 7892, -- All party members have fallen in battle. Now leaving the battlefield.
-        THE_PARTY_WILL_BE_REMOVED     = 7899, -- If all party members' HP are still zero after # minute[/s], the party will be removed from the battlefield.
-        YOU_NO_REQS                   = 7903, -- You do not meet the requirements to enter the battlefield with your party members. Access is denied.
-        YOUR_IMPERIAL_STANDING        = 7918, -- Your Imperial Standing has increased!
-        FEEL_A_CHILL                  = 8378, -- You feel a chill run down your spine!
-        HAND_OVER_TO_IMMORTAL         = 8436, -- You hand over the % to the Immortal.
-        CANNOT_ENTER                  = 8455, -- You cannot enter at this time. Please wait a while before trying again.
-        AREA_FULL                     = 8456, -- This area is fully occupied. You were unable to enter.
-        MEMBER_NO_REQS                = 8460, -- Not all of your party members meet the requirements for this objective. Unable to enter area.
-        MEMBER_TOO_FAR                = 8464, -- One or more party members are too far away from the entrance. Unable to enter area.
-        KEYHOLE                       = 8547, -- Fragments of bone, claws, and teeth are scattered around the gate...
-        KEYS_SHATTER                  = 8548, -- The <item> shatters... The <item> shatters... The <item> shatters...
```

Comment:
These need to be added back in and IDs need to be checked with POL Utils to get the correct moved IDs.